test/code/plugins/VMInsightsDataCollector_subtest.rb

```diff
@@ -376,14 +376,15 @@ def test_get_filesystems_live
                 data = io.readlines
                 data.each { |line|
                     s = line.split(" ")
-                    expected[s[0]] = [ s[1], s[2].to_i, false ]
+                    ind = s[0].index('/', 1)
+                    key = s[0][ind+1..-1]
+                    omit "running in container" if expected.has_key? key
```

Comment:
Suggestion: `omit "#{key} is mounted multiple times. Running in a container." if expected.has_key? key`